autoPyTorch/pipeline/components/training/trainer/RowCutOutTrainer.py

```diff
@@ -36,34 +32,23 @@ def data_preparation(self, X: np.ndarray, y: np.ndarray,
             lam = 1
             return X, {'y_a': y_a, 'y_b': y_b, 'lam': lam}
 
-        size = X.shape[1]
-        indices = self.random_state.choice(range(1, size), max(1, np.int32(size * self.patch_ratio)),
-                                           replace=False)
-
-        if not isinstance(self.numerical_columns, typing.Iterable):
-            raise ValueError("{} requires numerical columns information of {}"
-                             "to prepare data got {}.".format(self.__class__.__name__,
-                                                              typing.Iterable,
-                                                              self.numerical_columns))
-        numerical_indices = torch.tensor(self.numerical_columns)
-        categorical_indices = torch.tensor([index for index in indices if index not in self.numerical_columns])
-
-        # We use an ordinal encoder on the categorical columns of tabular data
-        # -1 is the conceptual equivalent to 0 in a image, that does not
-        # have color as a feature and hence the network has to learn to deal
-        # without this data. For numerical columns we use 0 to cutout the features
-        # similar to the effect that setting 0 as a pixel value in an image.
-        X[:, categorical_indices.long()] = self.CATEGORICAL_VALUE
-        X[:, numerical_indices.long()] = self.NUMERICAL_VALUE
+        size: int = np.shape(X)[1]
+        indices = self.random_state.choice(
```

Comment:
`indices` -> `shuffled_indices` for the compatibility with row-cut-mix trainer

Reply:
masked_indices would be good I guess and is in between of both things

Reply:
I think having only indices is a bit confusing. Let's stick with mentioning that these are column indices. Also, cut is fine because that's also the name of the algorithm, although I don't feel too strongly about this point. But for consistency, we can stick with cut_column_indices as we are using in `RowCutMixTrainer`

Reply:
Sure then, I agree.